package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vali-ah-frontend",
+  "version": "1.0.0",
+  "description": "1kbIdeas - The one-stop shop for all the self-help ideas you need. =======",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "jest:junit": "jest --ci --runInBand --reporters=default --reporters=jest-junit"
```

Comment:
Please, what is the advantage of jest-JUnit reporter over the default jest reporter?

Reply:
This is for the jest junit reporter. To report to circleci